### ISSUE TITLE
Clarify issues with case insensitive digests resulting in repeated digests

### ISSUE THREAD

**Issue by zimeon** [OCFL/spec#461]
Consider https://github.com/OCFL/fixtures/tree/master/1.0/bad-objects/bad16_digest_repeated which has inventory:

```
{
  "digestAlgorithm": "sha512",
  "head": "v1",
  "id": "ark:123/abc",
  "manifest": {
    "cf83e1357eefb8bdf1542850d66d8007d620e4050b5715dc83f4a921d36ce9ce47d0d13c5d85f2b0ff8318d2877eec2f63b931bd47417a81a538327af927da3e": [
      "v1/content/empty1"
    ],
    "CF83E1357EEFB8BDF1542850D66D8007D620E4050B5715DC83F4A921D36CE9CE47D0D13C5D85F2B0FF8318D2877EEC2F63B931BD47417A81A538327AF927DA3E": [
      "v1/content/empty2"
    ]
  },
  "type": "https://ocfl.io/1.0/spec/#inventory",
  "versions": {
    "v1": {
      "created": "2019-01-01T02:03:04Z",
      "message": "Example with two empty files that thus have the same digest but are represented with different normalizations",
      "state": {
        "cf83e1357eefb8bdf1542850d66d8007d620e4050b5715dc83f4a921d36ce9ce47d0d13c5d85f2b0ff8318d2877eec2f63b931bd47417a81a538327af927da3e": [
          "empty1"
        ],
        "CF83E1357EEFB8BDF1542850D66D8007D620E4050B5715DC83F4A921D36CE9CE47D0D13C5D85F2B0FF8318D2877EEC2F63B931BD47417A81A538327AF927DA3E": [
          "empty2"
        ]
      }
    }
  }
}
```

the digests are actually the same but in both `manifest` and `state` one is in uppercase and the other in lowercase. This could be allowed (should be text explaining merge after normalization) or it could be forbidden (what I'd assumed, should be text explaining that different normalizations MUST not result in repeated digests)

**Comment by zimeon** [comment]
See #437 where we decided to keep case insensitivity

**Comment by awoods** [comment]
Given the previous decision (https://github.com/OCFL/spec/issues/437#issuecomment-606625847), do we want to add "text explaining merge after normalization", or do nothing?

**Comment by zimeon** [comment]
I'm inclined to require consistent normalization within a single `inventory.json` but I think we have to add explanation either way

**Comment by neilsjefferies** [comment]
As per Community Call 13/5 - digest should occur once within a menifest regarless of case

**Comment by zimeon** [comment]
Closed by #468 and #469
